data-fundamentals/README.md

```diff
@@ -0,0 +1,78 @@
+# Data and DataOps fundamentals
+
+Most projects involve some type of data storage, data processing and data ops. For these projects, as with all projects, we follow the general guidelines laid out in other sections around security, testing, observability, CI/CD etc.
+
+## Goal
+
+This goal of this section is to help guide on some of the specifics around how to apply these fundamentals to data projects or data portions of a project.
```

Comment:
**The** goal...

The sentence does not skimp on words. How about: "The goal of this section is to briefly describe how to apply the fundamentals to data heavy projects or portions of the project."

Reply:
fixed